pkg/sshd/lock_test.go

```diff
@@ -12,6 +13,10 @@ const (
 	callbackTimeout = 1 * time.Second
 )
 
+var (
+	gitError = errors.New("git recieve error")
```

Comment:
Typo: "receive". Also I notice the commit message has a few typos. Should be:
`fix(sshd): unlock the repository when gitreceive succeeds or fails`